tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>Suggest heading of: Create Nx smaller models from weight clustering. See the <a href="https://github.com/tensorflow/model-optimization/pull/388" target="_blank">pruning example</a> as a reference.</p><p> </p><p>Seems like a better way to organize this. I know I suggested the backend-independent part previously but it's easier to understand by putting the original, the clustered Keras, and the clustered TFLite in one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<blockquote>Suggest heading of: Create Nx smaller models from weight clustering</blockquote><p>Do you mean we want all the headings look like the ones in pruning example or just this one?</p><p> </p>